npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater_spec.rb

```diff
@@ -2090,6 +2120,8 @@
             expect(updated_files.map(&:name)).
               to match_array(%w(package-lock.json packages/package1/package.json))
             expect(parsed_root_lockfile.dig("dependencies", "etag", "version")).to eq("1.8.1")
+            # Doesn't install etag as a top-level dependency
+            expect(parsed_root_lockfile.dig("packages", "", "dependencies", "etag")).to be_nil
```

Comment:
This isn't the most robust spec, would be better to add a updated lockfile fixture and expect the content matches.